templates/cisco_nxos_show_ip_arp_detail.textfsm

```diff
@@ -1,8 +1,14 @@
 Value ADDRESS (\d+\.\d+\.\d+\.\d+)
-Value AGE (\d+:\d+:\d+|-)
-Value MAC (\S+)
+Value AGE (-|(\d+(:|\.)\S+))
+Value MAC (INCOMPLETE|(\S{4}\.\S{4}\.\S{4}))
```

Comment:
could both of these just be `\S+`?
```suggestion
Value AGE (\S+))
Value MAC (\S+)
```

Reply:
Yes tested it works with with less specific match.